client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx

```diff
@@ -120,11 +120,11 @@ class CustomNameserversForm extends React.PureComponent {
 							onClick={ this.handleSubmit }
 							disabled={ this.props.submitDisabled }
 						>
-							{ translate( 'Save Custom Name Servers' ) }
+							{ translate( 'Save custom name servers' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.